examples/seekcamera-opencv_record.py

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env python3
```

Comment:
It is out of the scope of this pull request, but I think it makes sense to consolidate the changes in this file into `examples/seekcamera-opencv.py`. The recording changes are helpful and don't overload the purpose of this file in my opinion.